packages/snaps-sdk/src/jsx/components/form/Option.ts

```diff
@@ -9,7 +12,7 @@ import { createSnapComponent } from '../../component';
  */
 type OptionProps = {
   value: string;
-  children: string;
+  children: StringElement;
```

Comment:
Hmm, wondering if we want to support this 🤔

Reply:
Will revert